crates/oci/Cargo.toml

```diff
@@ -11,7 +11,7 @@ dkregistry = { git = "https://github.com/camallo/dkregistry-rs", rev = "37acecb4
 docker_credential = "1.0"
 dirs = "4.0"
 futures-util = "0.3"
-oci-distribution = { git = "https://github.com/krustlet/oci-distribution", rev = "64986855ef0d692df3b270d23c4bee8c41d97c27" }
+oci-distribution = { git = "https://github.com/fermyon/oci-distribution", branch = "access-token-cache" }
```

Comment:
We should pin to specific revs generally, and I'd really like those to be on `main` (or at least a protected branch) so they aren't accidentally deleted.

Reply:
The `access-token-cache` branch is now protected; updated to use a specific revision.